esi/lib/node.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I have a slight preference towards calling this `nodes.py`, since running `node.network_list` feels like it should only return information for one node.